frontend/web/components/modals/CreateFlag.js

```diff
@@ -242,10 +244,13 @@ const CreateFlag = class extends Component {
 
   fetchReleasePipelines = () => {
     if (!this.props.projectId) return
-
-    getReleasePipelines(getStore(), {
-      projectId: this.props.projectId,
-    })
+    getReleasePipelines(
+      getStore(),
+      {
+        projectId: this.props.projectId,
+      },
+      { forceRefetch: true },
```

Comment:
@Zaimwa9  I'm not sure if you experienced this before, but the value returned from the api is the most recent data but RTK is returning a stale data. For instance, I add a feature to a pipeline and a new GET /release-pipelines/ response from the api includes that feature in the response but RTK still returns the stale data that does not include that feature (`pipeline[n].features`)